README.md

```diff
@@ -25,25 +25,30 @@ Here is why I am rewriting a new container runtime in Rust.
 - The development of [railcar](https://github.com/oracle/railcar) has been suspended. This project was very nice but is no longer being developed. This project is inspired by it.
 - I have fun implementing this. In fact, this may be the most important.
 
+# Related project
+
+- [containers/oci-spec-rs](https://github.com/containers/oci-spec-rs) - OCI Runtime and Image Spec in Rust
+
 # Status of youki
 
 youki is not at the practical stage yet. However, it is getting closer to practical use, running with docker and passing all the default tests provided by [opencontainers/runtime-tools](https://github.com/opencontainers/runtime-tools).
 ![youki demo](docs/demo.gif)
 
 ## Features
 
-- [x] run with docker
-- [ ] run with podman(WIP on [#24](https://github.com/containers/youki/issues/24))
-- [x] pivot root
-- [x] mount devices
-- [x] namespaces
-- [x] capabilities
-- [x] rlimits
-- [x] cgroups v1
-- [ ] cgroups v2(WIP on [#78](https://github.com/containers/youki/issues/78))
-- [ ] seccomp(WIP on [#25](https://github.com/containers/youki/issues/25))
-- [ ] hooks(WIP on [#13](https://github.com/containers/youki/issues/13))
-- [ ] rootless(WIP on [#77](https://github.com/containers/youki/issues/77))
+|   Feature    |                   Description                   |                                                State                                                |
```

Comment:
Youki nowadays passes much more of the oci runtime tests. You could add it to the table with the number of tests that youki currently passes or update the 'Status of youki' section.

Reply:
I thought too many would be hard to see, so I narrowed it down to a few. I can still add a fe2w more, so do you have any recommendations for a feature.

Reply:
|   Feature    |                   Description                   |                                                State                                                |
| :----------: | :---------------------------------------------: | :-------------------------------------------------------------------------------------------------: |
|    OCI Compliance    |               Compliance with [OCI Runtime Spec  ](https://github.com/opencontainers/runtime-spec)             |                                                  X out of Y test cases passing                                               |

Reply:
@Furisto PTAL